packages/react-dev-utils/WebpackDevServerUtils.js

```diff
@@ -227,14 +230,14 @@ function createCompiler({
         messages.errors.length = 1;
       }
       console.log(chalk.red('Failed to compile.\n'));
-      console.log(messages.errors.join('\n\n'));
+      console.log(messages.errors.join('\n\n').replace('//', ''));
```

Comment:
What is this and the following `replace()` for? The regex doesn't look right but I am low on coffee.

Reply:
This is also related to path prefix stripping - both the console and the Error Overlay are driven from the same information.  This removes a double `//` from the console that would otherwise be printed.   Alas the `//` is necessary to make the Error Overlay display the colon as expected, eg `C:/`

See my other comment but I'm now of the opinion that path prefix stripping might not be a good idea.  I'd be happy to back that out.